fgd/bases/TeleTrigger.fgd

```diff
@@ -25,30 +25,21 @@
 	landmark(target_destination) : "Local Destination Landmark" : : "In landmark mode, teleported entities are offset from the target by " + 
 		"their initial offset from the landmark. Angles and velocity are also reoriented to the destination if enabled."
 	
-	velocityscale(vector) : "Velocity Scale Factor (X Y Z)" : "1 1 1" : "Scale the player's velocity per-axis"
-	
-	mode[engine](integer) : "Teleport mode" : 1
-	mode(choices) : "Teleport mode" : 1 = 
+	uselandmarkangles(boolean) : "Use Destination Angles" : 0 : "Forces the teleported object to snap to the angles of the destination."
+
+	checkdestifclearforplayer[-MOMENTUM](boolean) : "Only Spawn-Valid Destinations" : 0 : "Only teleport if there is a clear teleport destination."
+
+	TeleportOnEndTouch(boolean) : "Teleport On End Touch" : 0 : "Teleport on end touch instead of start touch"
```

Comment:
I still prefer framing this as a two-choice option instead of a boolean!
```
TeleportOnEndTouch(choices) : "Teleport On" : 0 = 
		[
		0 : "Start Touch"
		1 : "End Touch"
		]
```
IMO this makes it more intuitive that "end touch" is exclusive with "start touch", and that the default is "start touch". It's also the same UX changing the option, and the game code can still just treat it as a boolean.